src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Dataplane/Commands/AddAzureASAccount.cs

```diff
@@ -33,14 +33,49 @@ namespace Microsoft.Azure.Commands.AnalysisServices.Dataplane
     [OutputType(typeof(AsAzureProfile))]
     public class AddAzureASAccountCommand : AzurePSCmdlet, IModuleAssemblyInitializer
     {
-        [Parameter(Position = 0, Mandatory = false, HelpMessage = "Name of the Azure Analysis Services environment to which to logon to")]
+        private const string UserParameterSet = "UserParameterSetName";
+        private const string ServicePrincipalWithPasswordParameterSet = "ServicePrincipalWithPasswordParameterSetName";
+        private const string ServicePrincipalWithCertificateParameterSet = "ServicePrincipalWithCertificateParameterSetName";
+
+        [Parameter(ParameterSetName = UserParameterSet,
+            Mandatory = true, HelpMessage = "Name of the Azure Analysis Services environment to which to logon to", Position = 0)]
```

Comment:
Actually this  RolloutEnvironment is required even in previous version, they just forget to mark it as required.